packages/e2e-tests/recover/seedPhrase.spec.js

```diff
@@ -0,0 +1,45 @@
+const { test, expect } = require("@playwright/test");
+
+const { createRandomBankSubAccount } = require("../utils/account");
+
+const { describe, beforeAll, afterAll } = test;
+
+describe("Account Recovery Using Seed Phrase", () => {
+    let testAccount;
+
+    beforeAll(async () => {
+        testAccount = await createRandomBankSubAccount();
+    });
+    
+    afterAll(async () => {
+        testAccount && (await testAccount.delete());
+    });
+
+    test("navigates to seed phrase page successfully", async ({ page }) => {
+        await page.goto("/");
+
+        await page.click(
+            `button:text-matches("Import Existing Account", "i")`
+        );
+        await page.click(`data-test-id=recoverAccountWithPassphraseButton`);
+
+        expect(page).toMatchURL(/\/recover-seed-phrase$/);
+    });
+
+    test("Account Recovery Using Seed Phrase", async ({ page }) => {
+        await page.goto("/recover-seed-phrase");
+
+        await page.fill(
+            "data-test-id=seedPhraseRecoveryInput",
+            testAccount.seedPhrase
+        );
+        await page.click(`[type="submit"]`);
+
+        await page.waitForNavigation();
+
+        expect(page).toMatchURL(/\/$/);
+        expect(await page.textContent("data-test-id=currentUser")).toBe(
```

Comment:
Because we've added playwright matchers, I believe that this should be done using `.toMatchText()`... e.g.
```await expect(page).toMatchText("data-test-id=currentUser", testAccount.accountId)```

`await`'ing _inside_ of the `expect()` arguments here will cause errors thrown from `page.textContent()` to bubble up directly to the test runner, which will show a generic stack trace/failure instead of a useful `expect()`-created message -- this is because effectively, this line of code transpiles down to something like this:

```javascript
const textContent = await page.textContent("data-test-id=currentUser");
expect(textContent).toBe(testAccount.account.accountId);
```

See the issue? If the first line `throw`s, the second line won't even run so there's no way for `expect()` to create a user-friendly assertion error.

Reply:
Was using this to circumvent visibility issue since we have a hidden component for desktop with the same `data-test-id` but changed it to use a chained selector instead 👍